tools/wasm-build

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+base_dir=$(cd `dirname $0`; pwd)
+src_dir=${base_dir}/..
+build_folder=${src_dir}/wasm-build
+boost_dir=${EMSDK}/upstream/emscripten/system/include
```

Comment:
The variable EMSDK was not set on my system, after running the install-wasm-dependencies script.  Setting is manually worked.